src/poster.js

```diff
@@ -5,4 +5,6 @@ class Poster {
     this.title = title;
     this.quote = quote;
   }
-}
\ No newline at end of file
+}
+
+module.exports = Poster;
```

Comment:
Not sure if we need module.exports for this project but let's see what Cass says